test/integration/functional_test.go

```diff
@@ -573,6 +578,33 @@ func validateMySQL(ctx context.Context, t *testing.T, profile string) {
 	}
 }
 
+// Setup extra file for file sync test (validateFileSync), this
+// need to be done prior to starting up minikube.
+func validateCopySynFile(ctx context.Context, t *testing.T, profile string) {
+	// 1. copy random file to MINIKUBE_HOME/files/etc
+	tempfile := filepath.Join(localpath.MiniPath(), "/files/etc/filesync.txt")
+	err := copy.Copy("./testdata/filesync.txt", tempfile)
+	if err != nil {
+		t.Fatalf("copy: %v", err)
+	}
+}
+
+// validateFileSync start up VM and test sync file between host and VM.
+// On startup Minikube should copy the filesync.txt to /etc folder
+func validateFileSync(ctx context.Context, t *testing.T, profile string) {
+	// check file existence
+	statusCmd := "cat /etc/filesync.txt"
+	rr, err := Run(t, exec.CommandContext(ctx, Target(), "-p", profile, "ssh", statusCmd))
+	if err != nil {
+		t.Errorf("%s failed: %v", rr.Args, err)
+	}
+
+	// 'no' result means sync did not happen
```

Comment:
Remove this comment: the test failure message should always provide enough context.